HorizontalPodAutoscaler/README.md

```diff
@@ -0,0 +1,26 @@
+See:
```

Comment:
move refs to the end. this doc should be self-standing, with further reading at the end.